components/AppBar/src/MDCAppBarContainerViewController.m

```diff
@@ -45,11 +45,16 @@ - (void)viewDidLoad {
   [_appBar addSubviewsToParent];
 
   [_appBar.navigationBar observeNavigationItem:_contentViewController.navigationItem];
+
+  [self updateTopLayoutGuideBehavior];
 }
 
 - (void)viewWillLayoutSubviews {
   [super viewWillLayoutSubviews];
-  [_appBar.headerViewController updateTopLayoutGuide];
+
+  if (!self.topLayoutGuideAdjustmentEnabled) {
```

Comment:
Why not perform the adjustment if the feature is enabled?  We do another check inside the method, so I'm not sure why this has a gate added.

Reply:
Layout guide adjustment is now handled by the flexible header view controller, so this legacy logic is no longer necessary when topLayoutGuideAdjustmentEnabled is enabled.

Reply:
Behavior that occurs when `topLayoutGuideAdjustmentEnabled` is not true (the default behavior).

Reply:
Got it, thanks!